services/member-management-client/src/members-service.js

```diff
@@ -0,0 +1,14 @@
+const SERVICE_URL = 'http://127.0.0.1:3000';
```

Comment:
NAB: Once the dockerized client and users service ships, we should come back and replace hardcoded service urls with env vars.